commands/run/__tests__/run-command.test.js

```diff
@@ -217,23 +217,45 @@ describe("RunCommand", () => {
       const [logMessage] = loggingOutput("warn");
       expect(logMessage).toMatch("Dependency cycles detected, you should fix these!");
       expect(logMessage).toMatch("package-cycle-1 -> package-cycle-2 -> package-cycle-1");
-      expect(logMessage).toMatch("package-cycle-2 -> package-cycle-1 -> package-cycle-2");
-      expect(logMessage).toMatch(
-        "package-cycle-extraneous -> package-cycle-1 -> package-cycle-2 -> package-cycle-1"
-      );
 
       expect(output.logged().split("\n")).toEqual([
         "package-dag-1",
         "package-standalone",
         "package-dag-2a",
         "package-dag-2b",
-        "package-dag-3",
         "package-cycle-1",
         "package-cycle-2",
         "package-cycle-extraneous",
+        "package-dag-3",
```

Comment:
This should be ok: "dag" and "cycle" are two separate graphs, so there isn't a specified order between them.